packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -1230,7 +1280,8 @@ PlatformMapConfiguration _platformMapConfigurationFromOptionsJson(
     trafficEnabled: options['trafficEnabled'] as bool?,
     buildingsEnabled: options['buildingsEnabled'] as bool?,
     liteModeEnabled: options['liteModeEnabled'] as bool?,
-    cloudMapId: options['cloudMapId'] as String?,
+    markerType: _platformMarkerTypeFromIndex(options['markerType'] as int?),
```

Comment:
Do we actually need this helper method? I would not expect anything that supports marker types to call the deprecated method, so `markerType` should never be written.

Is there any code in any of our packages that would ever write this? If not, we can just hard-code it to `.marker`.